src/components/ExchangeRates/ExchangeRates.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery} from "@apollo/client";
+import { EXCHANGE_RATES } from '../../queries/queries';
```

Comment:
Hope someone eventually goes back to the history of this and sees us using the coinbase api to test this lol